codewof/programming/content/en/pass-the-parcel/question.md

```diff
@@ -0,0 +1,7 @@
+# Pass the parcel
+
+The following function `pass_the_parcel(parcel)` takes a list of strings (each item in the parcel), and should **print** `Person 1 got X!`, where `X` is the first item.
+The program should continue **printing** `Person n got X!` (where `n` is the nth person to receive something) until the end of the list.
```

Comment:
It's probably fine, but `X` is defined as the first item, not each item in sequence